security/src/main/java/io/micronaut/security/endpoints/LogoutControllerConfigurationProperties.java

```diff
@@ -43,8 +49,38 @@
     @SuppressWarnings("WeakerAccess")
     public static final String DEFAULT_PATH = "/logout";
 
+    /**
+     * Default Allowed Http Method.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static final HttpMethod DEFAULT_HTTPMETHOD = HttpMethod.POST;
+
+    private static final List<HttpMethod> SUPPORTED_ALLOWEDMETHODS = new ArrayList<HttpMethod>() {{
```

Comment:
I don't think anyone cares if logout is available via POST. I suggest changing this to a much simpler solution to something like `boolean allowGet`.